hawc/apps/study/forms.py

```diff
@@ -248,4 +251,6 @@ def get_query(self):
             query &= Q(**{data_type: True})
         if published:
             query &= Q(published=True)
+        if identifier:
```

Comment:
@casey1173 let's try out the walrus `:=` for these; this option didn't exist prior to python 3.8

https://realpython.com/lessons/assignment-expressions/